lumibot/backtesting/backtesting_broker.py

```diff
@@ -498,8 +498,8 @@ def process_pending_orders(self, strategy):
             # Get OHLCV data for the asset
             #############################
 
-            # Get the OHLCV data for the asset if we're using the YAHOO data source
-            if self.data_source.SOURCE == "YAHOO":
+            # Get the OHLCV data for the asset if we're using the YAHOO, CCXT data source
+            if self.data_source.SOURCE.upper() in ["CCXT","YAHOO"]:
```

Comment:
i don't believe this line makes sense. we do not use CCXT as a backtesting broker

Reply:
I have created a class for Binance backtesting using ccxt and am testing it. Currently, it only supports `spot`.

The `SOURCE` of the backtesting class for cryptocurrency is set to `SOURCE="CCXT"`. 
I added it because I think it would be good to use the same `SOURCE` as "CCXT" for other exchanges that use ccxt.

I haven't committed yet, but the structure of the backtesting class for Binance is as follows.

class BinanceDataBacktesting(BinanceData)


class BinanceData(DataSourceBacktesting):
    # SOURCE must be `CCXT` for the DataSourceBacktesting to work
    # `CCXT` is used in DataSource class
    SOURCE = "CCXT"
    MIN_TIMESTEP = "day"
    TIMESTEP_MAPPING = [
        {"timestep": "minute", "representations": ["1m"]},
        {"timestep": "day", "representations": ["1d"]},
    ]

Reply:
this sounds like an amazing addition. would you like some help with this? i'd be happy to get on a zoom call to see if i can help in any way. i can also introduce you to our other developers if you have questions. let me know, i'm happy to help